charts/kubernetes-zfs-provisioner/Chart.yaml

```
@@ -14,7 +14,7 @@ description: Dynamic ZFS persistent volume provisioner for Kubernetes
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.0.1
+version: 2.0.2
```

Since this is a new feature, I suggest a new minor release
```suggestion
version: 2.1.0
```